site/ar/tutorials/text/nmt_with_attention.ipynb

```diff
@@ -53,7 +53,7 @@
       "source": [
         "<table class=\"tfo-notebook-buttons\" align=\"right\" dir=\"rtl\">\n",
         "  <td>\n",
-        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/text/nmt_with_attention\">\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/text/nmt_with_attention?hl=ar\">\n",
```

Comment:
The Arabic version should automatically get served based on browser locale or the dropdown setting. This doesn't hurt, but probably not necessary for all the docs